numba/lowering.py

```diff
@@ -247,7 +257,13 @@ def lower_inst(self, inst):
             return impl(self.builder, (target, index, value))
 
         elif isinstance(inst, ir.Del):
-            pass
+            try:
+                # XXX: incorrect Del injection?
+                val = self.loadvar(inst.value)
```

Comment:
Yes, `getvar` will be enough.  The extra load wouldn't be a problem.  LLVM will optimize it out.  Ultimately, I want to improve the `Del` injection so I don't need the try-except here.